Doc/library/typing.rst

```diff
@@ -1023,7 +1023,7 @@ The module defines the following classes, functions and decorators:
 
 .. function:: NewType(typ)
```

Comment:
It looks like the function signature for `NewType` doesn't match the examples since they call it with two parameters, but there's only one here.  Can you also add that in this patch?  Thanks!

Reply:
Done! @csabella

Reply:
And thanks @csabella for your keen eye!